core/templates/dev/head/pages/exploration_editor/feedback_tab/feedback_tab.html

```diff
@@ -54,8 +54,8 @@ <h2>Exploration Feedback</h2>
     </div>
 
     <div class="row" ng-if="activeThread.messages">
-      <div style="margin-top: 10px; word-break: break-all;" class="col-lg-12 col-md-12 col-sm-12">
-        <table class="table">
+      <div style="margin-top: 10px" class="col-lg-12 col-md-12 col-sm-12">
+        <table class="table" style="table-layout: fixed; word-wrap: break-word;">
```

Comment:
Why do we need `table-layout: fixed` ? 

Also, a general question -- doesn't converting `word-break:break-all` to `word-wrap: break-word` fix the purpose ?

Reply:
@bansalnitish
If ```table-layout: fixed``` is not used, content flows out of the div.
In tables, ```word-wrap: break-word``` doesn't work.
Thanks for the review. :)

Reply:
Can you please add a screenshot for the same ?

Reply:
@bansalnitish I think it is working simply by changing `word-break: break-all` to `word-wrap: break-word`
Thanks. Making the change.

Reply:
@bansalnitish done! PTAL. :)